modules/efk-logging-collector-fluentd-v-rsyslog.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-fluentd.adoc
+
+[id="efk-logging-collector-fluentd-v-rsyslog_{context}"]
+= About Fluentd and Rsyslog
+
+You can choose between Fluentd and Rsyslog as the logging collector. 
+
+Some of the noteable differences between the two collectors are:
+
+* Daching differences.
+
+* Deleted namespace
+
+* No throttling in Rsyslog. With Fluentd, an administrator can reduce the rate at which the logs are read in by Fluentd before being processed. This feature is not currently available with Rsyslog.
+
+* No secure forward in Rsyslog. With Fluentd, you can use the Fluentd the `secure-forward` plug-in to send a copy of its logs to an external log aggregator. This feature is not currently available with Rsyslog.
+
+* remote syslog
```

Comment:
Although we do not have "built-in" support for remote syslog forwarding in rsyslog (like we have in fluentd via the `REMOTE_SYSLOG_*` env. var. settings), users can edit the rsyslog configmap to add support for syslog log forwarding using the `omfwd` module (https://www.rsyslog.com/doc/v8-stable/configuration/modules/omfwd.html).  If sending logs to another rsyslog, users can also use the better `omrelp` module (https://www.rsyslog.com/doc/v8-stable/configuration/modules/omrelp.html)